src/autocomplete/autocompleter.ts

```diff
@@ -560,22 +647,58 @@ export class Autocompleter extends Application {
 
     this._lastPulledRowCount = results.length;
 
-    // uuid ok here despite typescript
-    this._lastPulledSearchResults = results.map((item)=>({ uuid: item.uuid, name: item.name, pages: null})) as SearchResult[];
+    this._lastPulledSearchResults = results.map((item)=>({ uuid: item.uuid, name: item.name})) as SearchResult[];
 
     return;
   }
 
+  private async _searchCompediums(maxResultCount: number, documentName: string): Promise<DocumentType11[]> {
+    let results = [] as DocumentType11[];
+    
+    // No need to do anything if there is no place for any result.
```

Comment:
I like this idea.  But it's currently matching on the collection name (ex. 'world.sample-actors').  That seems less intuitive than matching on the label ('Sample Actors').  Would the typical user even know how to find the collection name?  There's some risk that labels are duplicates, but that seems like something the user should have to deal with.

On the other hand, matching by the name lets you do things like 'world.*' in the settings, which is much more convenient than it would be when using labels.  Thoughts?